site/gatsby-site/blog/the-first-taxonomy-of-ai-incidents/index.es.mdx

```diff
@@ -38,18 +38,18 @@ Puede ayudar a corregir los sesgos de informes de incidentes incrustados en los
 - [Abrir una solicitud de extracción](https://github.com/PartnershipOnAI/aiid)
 
 ## Crédito y agradecimientos
 
-La taxonomía CSET y el desarrollo del sistema AIID asociado son la colaboración de [Zachary Arnold](https://cset.georgetown.edu/staff/zachary-arnold/), [Sean McGregor](https://seanbmcgregor.com/) , [Helen Toner](https://cset.georgetown.edu/staff/helen-toner/), [Alex Friedland](https://cset.georgetown.edu/staff/alex-friedland/), [Alexandru Muscă ](https://github.com/alexmcode), y un grupo rotatorio de estudiantes de [Georgetown](https://www.georgetown.edu/) encargados de llenar la taxonomía de los incidentes en la base de datos. Los comentarios sobre la taxonomía CSET deben dirigirse a Helen Toner (cset@georgetown.rom 547 publicaciones), y al admitir múltiples taxonomías para las cuales la taxonomía CSET es un ejemplo temprano. Las taxonomías AIID son colecciones flexibles de clasificaciones gestionadas por personas y organizaciones expertas. Las taxonomías son los medios por los cuales la sociedad trabaja colectivamente para comprender tanto los incidentes individuales como las [estadísticas a nivel de población] (https://incidentdatabase.ai/taxonomy/cset) para estas clasificaciones. Las taxonomías de incidentes de IA bien estructuradas y rigurosamente aplicadas tienen la capacidad de informar las prioridades de investigación y formulación de políticas para una IA más segura, así como ayudar a los ingenieros a comprender las vulnerabilidades y los problemas producidos por sistemas inteligentes cada vez más complejos.
+La taxonomía CSET y el desarrollo del sistema AIID asociado son la colaboración de [Zachary Arnold](https://cset.georgetown.edu/staff/zachary-arnold/), [Sean McGregor](https://seanbmcgregor.com/) , [Helen Toner](https://cset.georgetown.edu/staff/helen-toner/), [Alex Friedland](https://cset.georgetown.edu/staff/alex-friedland/), [Alexandru Muscă ](https://github.com/alexmcode), y un grupo rotatorio de estudiantes de [Georgetown](https://www.georgetown.edu/) encargados de llenar la taxonomía de los incidentes en la base de datos. Los comentarios sobre la taxonomía CSET deben dirigirse a Helen Toner (cset@georgetown.rom 547 publicaciones), y al admitir múltiples taxonomías para las cuales la taxonomía CSET es un ejemplo temprano. Las taxonomías AIID son colecciones flexibles de clasificaciones gestionadas por personas y organizaciones expertas. Las taxonomías son los medios por los cuales la sociedad trabaja colectivamente para comprender tanto los incidentes individuales como las [estadísticas a nivel de población](https://incidentdatabase.ai/taxonomy/cset) para estas clasificaciones. Las taxonomías de incidentes de IA bien estructuradas y rigurosamente aplicadas tienen la capacidad de informar las prioridades de investigación y formulación de políticas para una IA más segura, así como ayudar a los ingenieros a comprender las vulnerabilidades y los problemas producidos por sistemas inteligentes cada vez más complejos.
```

Comment:
External links do not seem to work. Clicking the Zachary Arnold one takes me to "http://localhost:8000/eshttps://cset.georgetown.edu/staff/zachary-arnold/"

Reply:
There are some extra spaces on [Line 17](https://github.com/responsible-ai-collaborative/aiid/blob/e6e120f6b47cddc2c5dc378d5fbac5942a5e2270/site/gatsby-site/blog/the-first-taxonomy-of-ai-incidents/index.es.mdx?plain=1#L17)

```
con el [Centro para la Seguridad y Tecnología Emergente] (https://cset.georgetown.edu/) (CSET) en Georgetown
                                                        ^
[...] para su [manual sobre accidentes de IA] (https://cset.georgetown.edu/publication/ai-accidents-an-emerging-threat/).
                                       ^
```

The link at [line 39](https://github.com/responsible-ai-collaborative/aiid/blob/e6e120f6b47cddc2c5dc378d5fbac5942a5e2270/site/gatsby-site/blog/the-first-taxonomy-of-ai-incidents/index.es.mdx?plain=1#L39) just doesn't work – I'm not sure if it's the length or the format, or what.

```
    - [Los producidos por organizaciones asociadas](/apps/discover?classifications=CSET%3ASystem%20Developer%3AAmazon%7C%7CCSET%3ASystem%20Developer%3AGoogle%7C%7CCSET%3ASystem%20Developer%3AMicrosoft%7C%7CCSET%3ASystem% 20Desarrollador%3AOpenAI%7C%7CCSET%3ASystem%20Desarrollador%3AFacebook%7C%7CCSET%3ASystem%20Desarrollador%3AYouTube)
```

Reply:
The cset@georgetown.rom gets turned into a link, but it's also interpreted as a relative link rather than a mailto. It should also be ".edu" rather than ".rom."